tests/vmi_lifecycle_test.go

```diff
@@ -261,9 +261,11 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 		DescribeTable("log libvirtd debug logs should be", func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
 			var err error
-			vmi := tests.NewRandomVMI()
-			vmi.Labels = vmiLabels
-			vmi.Annotations = vmiAnnotations
+			options := []libvmi.Option{libvmi.WithResourceMemory("32Mi")}
+			for k, v := range vmiLabels {
```

Comment:
I believe you should repeat this with `vmiAnnotations`

Reply:
Thanks forgot it.
Done.